package.json

```diff
@@ -62,8 +62,8 @@
     "jsdom": "^9.12.0",
     "mocha": "^4.0.0",
     "prop-types": "^15.6.0",
-    "react": "^15.6.1",
-    "react-dom": "^15.6.1",
+    "react": "^0.14 || ^15 || ^16",
+    "react-dom": "^0.14 || ^15 || ^16",
```

Comment:
@ljharb like this? when I test this it always installs v16

Reply:
yep! that's fine, it will always install the latest, but it will be *valid* to install older versions.